pom.xml

```diff
@@ -33,11 +33,11 @@
         <dice-provider-artifact-id>dice-provider</dice-provider-artifact-id>
         <x509-provider-artifact-id>x509-provider</x509-provider-artifact-id>
 
-        <iot-device-client-version>1.23.0</iot-device-client-version>
-        <iot-service-client-version>1.23.0</iot-service-client-version>
-        <iot-deps-version>0.9.5</iot-deps-version>
-        <provisioning-device-client-version>1.8.2</provisioning-device-client-version>
-        <provisioning-service-client-version>1.6.1</provisioning-service-client-version>
+        <iot-device-client-version>1.24.0</iot-device-client-version>
+        <iot-service-client-version>1.24.0</iot-service-client-version>
+        <iot-deps-version>0.9.6</iot-deps-version>
```

Comment:
the PR description says that this should be `0.10.0`?

Reply:
Good catch, I missed that one

Reply:
Now it's fixed